tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -2589,19 +2589,31 @@ public function waitForXSeconds($seconds): void {
 	}
 
 	/**
-	 * @When apply message expiration job to room :identifier
+	 * @When apply message expiration job
 	 */
-	public function applyMessageExpirationJobToRoom($identifier): void {
+	public function applyMessageExpirationJob(): void {
 		$currentUser = $this->currentUser;
 		$this->setCurrentUser('admin');
-		$this->sendRequest('GET', '/apps/spreedcheats/get_message_expiration_job/' . self::$identifierToToken[$identifier]);
+		$this->sendRequest('GET', '/apps/spreedcheats/get_message_expiration_job');
 		$response = $this->getDataFromResponse($this->response);
-		Assert::assertIsArray($response, 'Room ' . $identifier . 'not found');
-		Assert::assertArrayHasKey('id', $response, 'Job not found by identifier "' . $identifier . '"');
+		Assert::assertIsArray($response, 'Job not found');
+		Assert::assertArrayHasKey('id', $response, 'Job not found');
 		$this->runOcc(['background-job:execute', $response['id']]);
 		$this->setCurrentUser($currentUser);
 	}
 
+	/**
+	 * @When message expiration job don't exists
+	 */
+	public function messageExpirationJobDontExists(): void {
```

Comment:
```suggestion
	 * @When message expiration job doesn't exists
	 */
	public function messageExpirationJobDoesntExists(): void {
```

But then again, you can remove this when you hardcode the backgroundjob to always be there